src/key_packages/mod.rs

```diff
@@ -119,7 +119,7 @@ impl KeyPackage {
     }
 
     /// Compute the hash of the encoding of this key package.
-    pub(crate) fn hash(&self) -> Vec<u8> {
```

Comment:
The client needs to know the hash of a given `KeyPackage`, for example when communicating with the key store (or wherever the `KeyPackages` are uploaded`, so I think it's a good idea to expose it.